src/QueueManager.php

```diff
@@ -193,15 +193,13 @@ public static function engine(string $name): SimpleClient
      *      string 'default' if empty.
      * @return void
      */
-    public static function push($callable, array $args = [], array $options = []): void
+    public static function push(string $class, array $args = [], array $options = []): void
```

Comment:
This looks like a breaking change that will impact more users though. Making all existing configurations invalid seems like a user-hostile solution. Could we instead allow both the array form of `callable` and the new classname interface?

These parameter types allow us to easily convert the string form into the the array form with the code you added, and then also do the array type checks like in the code you're removed. Then in the future we can deprecate the old call format.

Reply:
I updated the method to accept either a string or a string array.